src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs

```diff
@@ -57,7 +57,15 @@ public override async Task RemoveAsync(string[] urls)
         public override async Task<BlobEntrySearchResult> SearchAsync(string folderUrl, string keyword)
         {
             folderUrl = NormalizeUrl(folderUrl);
-            return await base.SearchAsync(folderUrl, keyword);
+
+            var result = await base.SearchAsync(folderUrl, keyword);
```

Comment:
Can you rename `result` to something more informative?

Reply:
I think that `result` suitable for this place. You can see returning type in method signature.

Reply:
I think it's good as is